docs/userguide/storagedriver/imagesandcontainers.md

```diff
@@ -148,15 +142,6 @@ One way to do this is using a Dockerfile and the `docker build` command.
     The 4 image layers below it are the exact same image layers the make up the
     ubuntu:15.04 image as their UUIDs match.
```

Comment:
@nigelpoulton nothing has changed concerning where the layer data is located on disk. There were almost no changes to graphdrivers(except windows). Only thing that is changed is the addressing model from docker objects. I can provide some tools/scripts for people who want to explore the files on disk. There is also a (vague) plan to have something that visualizes the layer sharing, probably as an external tool at first.